### PR TITLE
add maker festival proposal

### PR DESCRIPTION
**Pull request by dcwalk**
Hey, I recall we spoke about this...for posterity we should put up a draft of our maker proposal, what do people think?

### CODE REVIEW

File: meeting_notes/20160430_proposal-notes.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        - hand switching
+    - topics for activities:
+        - transmission, addressing, routing/hops
+        - 'packets,' encoding
+        - 'topology,' node, links
+
+## Reference Links
+
+- [networks.land](http://networks.land/)
+- [github.com/mofarrell/p2pvc](https://github.com/mofarrell/p2pvc)
```
Comment (benhylau) [inline meeting_notes/20160430_proposal-notes.md:57]: 📹

File: meeting_notes/20160430_proposal-notes.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- Have a hands-on model explaining how mesh networks function. This interactive and playful station will encourage visitors of all ages and abilities to take part!
+- Provide scaled-down learning materials that serve as an introduction to our satellite event, where we will conduct interactive group activities to communicate basic networking concepts (Semaphore Research Cluster members have held on designing mesh networks, network concepts and digital literacy).
+
+## Describe your project BRIEFLY (50 words or less)
+
+_this smaller description may be used on signage at the festival_
+
+Join Toronto Mesh as we build a wireless community network. Learn how the internet and mesh technology works with tomesh.mini! See what's going on across the festival venue without the web. Visit us to interact with it and see a hands-on analogue to a mesh network.
+
+_Ben's revision:_
```
Comment (benhylau) [inline meeting_notes/20160430_proposal-notes.md:93]: Not sure which version was submitted `¯\(°_o)/¯`

**Comment by benhylau** [comment]
Thanks, totally forgot about this, looks good for merge

File: meeting_notes/20160430_proposal-notes.md (hunk 2 above)
Comment (dcwalk) [inline meeting_notes/20160430_proposal-notes.md:93]: yeah not sure that matters too much... both here for reference!

**Pull request merged**